### ISSUE TITLE
Uncaught Exception Raised When Indexing Notes

### ISSUE THREAD

**Issue by kwrobert** [nybbles/obsidian-pocket#83]
It looks like this plugin raises the following exception for me when installed through the Community Plugin browser in Obsidian: 

```
Uncaught (in promise) DOMException: Failed to execute 'transaction' on 'IDBDatabase': One of the specified object stores was not found.
    at Proxy.eval (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:30479:26)
    at URLToPocketItemNoteIndex.eval (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:30877:26)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:28:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:24:12)
    at URLToPocketItemNoteIndex.indexURLsForAllFilePaths (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:30868:43)
    at PocketSync.eval (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:32839:39)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app://obsidian.md/app.js:1:1362602), <anonymous>:28:71)
eval @ wrap-idb-value.js:150
eval @ URLToPocketItemNoteIndex.ts:138
eval @ tslib.es6.js:76
__awaiter @ tslib.es6.js:72
URLToPocketItemNoteIndex.indexURLsForAllFilePaths @ URLToPocketItemNoteIndex.ts:124
eval @ main.ts:169
eval @ tslib.es6.js:76
__awaiter @ tslib.es6.js:72
eval @ main.ts:168
Promise.then (async)
step @ tslib.es6.js:75
eval @ tslib.es6.js:76
__awaiter @ tslib.es6.js:72
eval @ main.ts:168
setTimeout (async)
eval @ main.ts:168
fulfilled @ tslib.es6.js:73
Promise.then (async)
step @ tslib.es6.js:75
fulfilled @ tslib.es6.js:73
Promise.then (async)
step @ tslib.es6.js:75
fulfilled @ tslib.es6.js:73
Promise.then (async)
step @ tslib.es6.js:75
eval @ tslib.es6.js:76
__awaiter @ tslib.es6.js:72
onload @ main.ts:73
t.load @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
t.loadPlugin @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
t.enablePlugin @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
t.enablePluginAndSave @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
(anonymous) @ app.js:1
e.setValue @ app.js:1
e.onClick @ app.js:1
```

Cloned the the repo to take a look, code, and I'm thinking maybe the local cache for Pocket data is never created in the Obsidian database? I'm using Obsidian v0.13.23 on MacOS. Let me know how I can help debug this further, would love to get this integration working!

**Comment by nybbles** [comment]
Thanks for raising this issue and sorry that you experienced it :( I replicated it and am working on a fix now.

**Comment by kwrobert** [comment]
Hey no problem! Thanks for jumping on this so quick. Let me know if there is anything I can do to help.

**Comment by nybbles** [comment]
hey this should be fixed now in 0.6.1. Please try it out and let me know if it works for you. Please reopen this issue if it does not work.
